doc/yaml_docs.go

```diff
@@ -149,6 +149,10 @@ func genFlagResult(flags *pflag.FlagSet) []cmdOption {
 	var result []cmdOption
 
 	flags.VisitAll(func(flag *pflag.Flag) {
+		if flag.Hidden {
```

Comment:
```suggestion
		if len(flag.Deprecated) > 0 || flag.Hidden { 
```

Let's do the same in the `man` doc generation